acceptance-tests/src/testFixtures/java/tech.pegasys.signers.secp256k1/DefaultFileSelector.java

```diff
@@ -33,4 +33,19 @@
       return entry.getFileName().toString().equals(filename + ".toml");
     };
   }
+
+  @Override
+  public Filter<Path> getSingleConfigFileFilter(final String address) {
+    // remove 0x if part of address
+    return entry -> {
+      final String fileName;
+      if (address.toUpperCase().startsWith("0X")) {
+        fileName = address.substring(2);
+      } else {
+        fileName = address;
+      }
```

Comment:
This looks like an improved version of what the getSpecificConfigFileFilter can we use the same code for both i.e. removing the 0x prefix if needed.